tests/src/lib_tests.rs

```diff
@@ -1,16 +1,17 @@
 use clar2wasm::compile;
```

Comment:
Why are the tests in the "src" folder now?

We really should get rid of this useless "clar2wasm-tests" package, it doesn't make any sense to have this.

Reply:
Yeah, this change was sort of a step in the direction of removing this separate tests package. It seemed strange to me to have tests/tests/ so I started off by moving everything into one directory, but ultimately I agree, this can all just go in the clar2wasm/tests/ directory. Let's finish that migration in its own PR.